Resources/views/show.html.twig

```diff
@@ -11,6 +11,9 @@
         $('#form-filter-{{ widget.id }} input').on('change', function(e) {
             $(e.target).parents('form').submit();
         });
+        $('#form-filter-{{ widget.id }} select').on('change', function(e) {
```

Comment:
something like this would be better don't you think ?:
```javascript
$(document).on('#form-filter-{{ widget.id }} input, #form-filter-{{ widget.id }} select', 'change', function(e) {
    $(e.target).parents('form').submit();
});
```